bfasst/flows.py

```diff
@@ -19,10 +19,19 @@
 from bfasst.opt.ic2_lse import Ic2LseOptTool
 from bfasst.opt.ic2_synplify import Ic2SynplifyOptTool
 from bfasst.reverse_bit.icestorm import IcestormReverseBitTool
-from bfasst.status import Status, MapStatus, CompareStatus, TransformStatus, ErrorInjectionStatus
+from bfasst.status import (
+    BfasstException,
+    Status,
+    MapStatus,
+    CompareStatus,
+    TransformStatus,
+    ErrorInjectionStatus
+)
 from bfasst.synth.ic2_lse import Ic2LseSynthesisTool
 from bfasst.synth.ic2_synplify import Ic2SynplifySynthesisTool
 from bfasst.synth.yosys import YosysTechSynthTool
+from bfasst.transform.error_injector import ErrorInjector as SdnErrorInjector
+from bfasst.compare.structural import StructuralCompareTool
```

Comment:
We try not to import the tools in flows.py (yes there are some above we haven't dealt with yet).  Can you add the imports inside your function instead?

Reply:
Pylint doesn't like the tools imported outside the top level in flows.py. I can do it that way, but it will break CI

Reply:
You can add this to flows.py: https://github.com/byuccl/bfasst/blob/main/bfasst/tool_wrappers.py#L4

Reply:
Done

Reply:
Looks like those two imports pushed me over the limit for the number of variables in my flow though. I don't know if trying to split the function makes it cleaner in the context of the flows module though. What are your thoughts?